lib/datapacktypes/vlocityuitemplate.js

```diff
@@ -0,0 +1,26 @@
+var async = require('async');
+
+var VlocityUITemplate = module.exports = function(vlocity) {
+	this.vlocity = vlocity;
+};
+
+VlocityUITemplate.prototype.onDeployFinish = async function(jobInfo) {
+    if (!this.vlocity.sfdxUsername) {
```

Comment:
And can you check to make sure this isn't running for every VlocityUITemplate deployment, but for the ones related to OmniScripts based on the Type of the VlocityUITemplate.

Reply:
And it should be controlled by a setting to turn off.